pkg/rtc/changenotifier.go

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
Can this be move to `pkg/utils` or some other common place please? Nothing specific to `rtc` here and potentially can be used by others if necessary.